### ISSUE TITLE
number_recognition.ino is not working

### ISSUE THREAD

**Issue by codenio** [sonydevworld/spresense-arduino-compatible#134]
Example **number_recognition.ino** is not producing intended results

![image](https://user-images.githubusercontent.com/30741615/144739833-c5121808-28e7-4d56-aaba-bda71fcb3a30.png)

Results are not being displayed and it stays in the same state after feeding the processed test image into NN
I have used a mnist-classifer from nnabla misnt-collection example. 

adding the relevant model file **network.nnb** file to this example repo would be helpful for the beginers 

help needed with debugging. thanks in advance.

**Comment by SPRESENSE** [comment]
@codenio Thanks for reporting issue.
There is an known issue of NNB format version dependency if you uses the [Neural Network Libraries](https://nnabla.org/install/).
Since the version of NNB currently supported by Spresense is old, you need to specify the API level to convert.
e.g.
~~~
nnabla_cli convert -b 1 --api 3 lenet_result.nnp network.nnb
~~~

Also, if you are using `misnt-collection` as is, change the `number_recognition.ino` sketch as below:
```diff
@@ -79,7 +80,7 @@ void setup() {

   for (int x = 0; x < height; x++) {
     for (int y = 0; y < width; y++) {
-      buf[i] = float(pgm.getpixel(x, y)) / 255.0;
+      buf[i] = float(pgm.getpixel(x, y)) / 1.0;
       i++;
     }
   }
```
</pre>

In the next SDKv2.4.0 version, we plan to support the latest version of the NNB format.
Please look forward to the next version.

**Comment by codenio** [comment]
@SPRESENSE thanks for the prompt response.

the code works as expected, It was really helpful :handshake:

![image](https://user-images.githubusercontent.com/30741615/145136303-cc1c91ae-8e8c-4523-8600-f158e9cc9975.png)

it will be helpful if we could add the working **network.nnb** file to this example folder.
do let me know if i could submit a PR for the same.

**Comment by SPRESENSE** [comment]
@codenio That's a good suggestion.
To make it easy for users to run `number_recognition.ino`, I will add an example `network.nnb` generated by Neural Network Console (NNC). This example sketch is for NNC, so it is different from the NNB generated by Neural Network Libraries (NNL).

**Comment by SPRESENSE** [comment]
Added  `network.nnb` generated by NNC on `develop` branch.

**Comment by SPRESENSE** [comment]
Released v2.4.0.
